run_project_tests.py

```diff
@@ -380,7 +380,12 @@ def run_tests(extra_args):
     build_time = 0
     test_time = 0
 
-    executor = conc.ProcessPoolExecutor(max_workers=multiprocessing.cpu_count())
+    try:
+        # This fails in some CI environments for unknown reasons.
+        num_workers = multiprocessing.cpu_count()
+    except Exception:
```

Comment:
Why not `Exception`? I tested this manually and it catches a `NotImplementedError` without problems as it inherits from `Exception`. Also, the error we get in Debian reproducible builds is not `NotImplementedError` but instead something else (related so semaphors?), but sadly I don't have a link to the error log handy.